source/js/main.js

```diff
@@ -29,9 +30,13 @@ document.addEventListener("DOMContentLoaded", function() {
     new MenuToggle(menutoggle, openMenu, closeMenu);
   }
 
+  for (const donatetoggle of document.querySelectorAll(
+    AmountToggle.selector()
+  )) {
+    new AmountToggle(donatetoggle);
+  }
+
   for (const tabs of document.querySelectorAll(Tabs.selector())) {
     new Tabs(tabs);
   }
-
-  tabIndexer();
```

Comment:
was this deleted on purpose as it looks like the function still exists? Or is it being called elsewhere?

Reply:
Good spot, shouldn't have been removed, thanks!